centreon-open-tickets/www/modules/centreon-open-tickets/providers/EasyVistaRest/EasyVistaRestProvider.class.php

```
@@ -90,13 +94,17 @@ public static function test($info): void
     public function closeTicket(&$tickets): void
     {
         if ($this->doCloseTicket()) {
-            foreach ($tickets as $k => $v) {
+            foreach ($tickets as $ticket => $v) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$v'. (undefined)

(UnusedLocalVariable)